feconf.py

```diff
@@ -505,6 +506,11 @@ def get_empty_ratings():
 
 # Unfinished features.
 SHOW_TRAINABLE_UNRESOLVED_ANSWERS = False
+# Number of unresolved answers to be displayed in the dashboard for each
+# exploration.
+TOP_UNRESOLVED_ANSWERS_COUNT_DASHBOARD = 3
```

Comment:
Is it possible at all to limit earlier? Unnecessary datastore fetches cost time/money, so it seems worth optimizing since the dashboard page is quite frequently used.